pandas/tests/frame/methods/test_join.py

```diff
@@ -575,3 +575,41 @@ def test_frame_join_tzaware(self):
 
         tm.assert_index_equal(result.index, expected)
         assert result.index.tz.key == "US/Central"
+
+    def test_frame_join_categorical_index(self):
+        # GH 61675
+        cat_data = pd.Categorical(
+            [15, 16, 17, 18],
+            categories=pd.Series(list(range(3, 24)), dtype="Int64"),
```

Comment:
Can you reduce the size here; even 20 elements can make stepping through with a debugger harder. Shoot for 3 or 5 if necessary.